### PR TITLE
DOCSP-37440: documentdb feedback on indexes

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-ecosystem/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-37440>
Staging - https://preview-mongodbrustagir.gatsbyjs.io/drivers/DOCSP-37440-index-documentdb/documentdb-support/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/documentdb-support.txt [hunk 1]
```diff
@@ -69,6 +69,16 @@ available in a limited capacity or are not supported at all:
   ``$toObjectId``, ``$toString``, ``$isNumber``, ``$rand``, ``$sampleRate``, ``$getField``
 - **Aggregation Stages**: ``$collStats``, ``$facet``, ``$bucket``, ``$bucketAuto``,
   ``$sortByCount``, ``$listLocalSessions``, ``$listSessions``, ``$graphLookup``
+  
+Amazon DocumentDB also does not leverage indexes when you perform
```
Comment (mongoKart) [inline source/documentdb-support.txt:73]: S: "also does not" makes it sound like this is building on a previous list of times DocumentDB doesn't use indexes.

```suggestion
In addition, Amazon DocumentDB does not leverage indexes when you perform
```

File: source/documentdb-support.txt [hunk 2]
```diff
@@ -69,6 +69,16 @@ available in a limited capacity or are not supported at all:
...
+queries that contain any of the following operators:
```
Comment (mongoKart) [inline source/documentdb-support.txt:74]: S: ticket mentions this too

```suggestion
case-sensitive queries or queries that contain any of the following operators:
```

**Review by mongoKart — APPROVED** [review]
lgtm + 2 suggestions. can you run this by Alex Bevilacqua too?

File: source/documentdb-support.txt (hunk 2 above)
Comment (rustagir) [reply source/documentdb-support.txt:74]: this is already listed as a non compatible feature in the preceding list!

**Review by alexbevi — APPROVED** [review]
LGTM

**Pull request merged**